keyboards/kopibeng/xt87/readme.md

```diff
@@ -0,0 +1,25 @@
+# XT87
+
+![XT87](https://i.imgur.com/N1QaXPTl.jpg)
```

Comment:
```suggestion
![XT87](https://i.imgur.com/N1QaXPTlh.jpg)
```

Reply:
Changed imgur link to huge thumbnail from large.